pkg/router/template/router.go

```diff
@@ -99,7 +100,9 @@ type templateData struct {
 	// the directory that files will be written to, defaults to /var/lib/containers/router
 	WorkingDir string
 	// the routes
-	State map[string]ServiceUnit
+	State map[string](ServiceAliasConfig)
+	// the service lookup
+	ServiceUnits map[string]ServiceUnit
```

Comment:
This makes it even harder to refactor to use route objects internally, which concerns me because we need to be moving in that direction.